src/main/java/org/cbioportal/cdd/service/util/MskVocabularyConverter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+package org.cbioportal.cdd.service.util;
+
+import java.util.*;
+import org.cbioportal.cdd.model.ClinicalAttributeMetadata;
+import org.cbioportal.cdd.model.MskVocabularyField;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MskVocabularyConverter {
+
+    public ClinicalAttributeMetadata convertToClinicalAttributeMetadata(MskVocabularyField mskVocabularyField) {
+        ClinicalAttributeMetadata clinicalAttributeMetadata = new ClinicalAttributeMetadata(
+            mskVocabularyField.getVariable().toUpperCase(),
+            mskVocabularyField.getLabel(),
+            mskVocabularyField.getLabel(),
```

Comment:
Here we are hardcoding priority to 1 and description to hold the display_name value. We will be soon updating this when the upstream system is fully functional.